src/main/java/com/tannat/country/domain/Country.java

```diff
@@ -0,0 +1,16 @@
+package com.tannat.country.domain;
+
+import lombok.Data;
```

Comment:
I didn't see lombok dependency in build gradle

Reply:
here https://projectlombok.org/setup/gradle  they write "There is a plugin for gradle that we recommend you use", so i added it